tests/Microsoft.Bot.Builder.Ai.LUIS.Tests/LuisRecognizerTests.cs

```diff
@@ -297,21 +284,21 @@ private JObject Json<T>(T result)
         // 1) Create a <name>.json file with an object { Text:<query> } in it.
         // 2) Run this test which will fail and generate a <name>.json.new file.
         // 3) Check the .new file and if correct, replace the original .json file with it.
-        public async Task TestJson<T>(string file)
-            where T : IRecognizerConvert, new()
+        public async Task TestJson<T>(string file) where T : IRecognizerConvert, new()
         {
-            if (!EnvironmentVariablesDefined())
-            {
-                Assert.Inconclusive("Missing Luis Environment variables - Skipping test");
-                return;
-            }
-
-            var expectedPath = Path.Combine(@"..\..\..\TestData\", file);
+            var expectedPath = GetFilePath(file);
+            var mockPath = GetFilePath("Mock_" + file);
             var newPath = expectedPath + ".new";
-            var luisRecognizer = GetLuisRecognizer(verbose: true, luisOptions: new LuisRequest { Verbose = true });
+
             var expected = new StreamReader(expectedPath).ReadToEnd();
             dynamic expectedJson = JsonConvert.DeserializeObject(expected);
```

Comment:
Not sure why this is `dynamic` here... it's very clearly only used as a `JToken`.